code/game/objects/items/ego_weapons/_ego_weapon.dm

```diff
@@ -24,7 +24,8 @@
 /obj/item/ego_weapon/examine(mob/user)
 	. = ..()
 	. += EgoAttackInfo(user)
-	. += "<span class='notice'>[special]</span>"
+	if(special != "This weapon has no special abilities.")	//This is honestly a failsafe for if it doesn't go through. A failsafe debug text, if you would
```

Comment:
Why not just make special default to null..? :thinking:

Reply:
Because the "No abilities" text works to make them have 4 lines by default